src/modules/playRoom/pages/PlayRoom.scss

```diff
@@ -0,0 +1,124 @@
+
+.noti {
+    margin-top: 30px;
+    width: 100%;
+    height: 570px;
+    display: flex;
+    justify-content: center;
+    &-box {
+        width: 100%;
+        height: 100%;
+        background-color: $bg-color-box;
+        overflow: hidden;
+        border-radius: 10px;
+        position: relative;
+        &-client {
+            display: flex;
+            justify-content: start;
+            align-items: center;
+            height: 70px;
+            background-color: $bg-color-box-client;
+            padding: 15px;
+            &-info {
+                padding: 5px;
+                margin-top: 15px;
+                h2 {
+                    font-weight: 650;
+                    color: $text-color;
+                }
+            }
+        }
+        .scroll {
+            width: 100%;
+            height: 74%;
+            overflow: hidden;
+            overflow-y: scroll;
+            .chats {
+                width: 100%;
+                padding: 0 15px;
+                color: $text-color;
+                position: relative;
+                font-size: 0.9em;
+                height: 100%;
+                &-client {
+                    width: 70%;
+                    word-wrap: break-word;
+                    background-color: $bg-color-chat-client ;
+                    padding: 7px 10px;
+                    border-radius: 10px 10px 10px 0;
+                    margin: 10px 0;
+                }
+                &-self {
+                    width: 65%;
+                    word-wrap: break-word;
+                    background-color: $bg-color-chat-self ;
+                    padding: 7px 10px;
+                    border-radius: 10px 10px 0 10px;
+                    margin: 5px 0 5px auto;
+                }
+            }
+        }
+        &-form {
+            display: flex;
+            align-items: center;
+            width: 100%;
+            height: 65px;
+            background-color: $bg-color-box-form;
+            padding: 15px;
+            overflow: hidden;
+            position: absolute;
+            bottom: 0;
+            input{ 
+                width: calc(100% - 50px);
+                height: 100%;
+                background-color: $bg-color-box-form-input;
+                border-radius: 50px;
+                color: $text-color;
+                font-size: 1.2em;
+                padding: 0 15px;
+                border:  transparent;
+            }
+            &-btn-send {
+                width: 40px;
+                height: 40px;
+                background-color: transparent;
+                position: relative;
+                margin-left: 5px;
+                cursor: pointer;
+                transition: 0.4s ease-in-out;
+                border:  transparent;
+                img {
+                    width: 40px;
+                    height: 40px;
+                    margin-right: 12px;
+                }
+            }
+        }
+    }
+}
+
+::-webkit-scrollbar{
```

Comment:
Please understand that you are trying to modify all the scrollbars that can appear on our website